modules/schedule_module/jsx/scheduleIndex.js

```diff
@@ -0,0 +1,565 @@
+   *
+   * @return {*} a formated table cell for a given column
+   */
+  formatColumn(column, cell, row) {
+    let result = <td>{cell}</td>;
+    switch (column) {
+    case 'Edit':
+      result = <td><button className="btn btn-default" onClick={() => this.edit(row)}><span className="glyphicon glyphicon-edit"></span> Edit</button></td>;
+      break;
+    case 'Delete':
+      result = <td><button className="btn btn-default" onClick={() => this.deleteid(row['Delete'])}><span className="glyphicon glyphicon-trash"></span> Delete</button></td>;
+      break;
+    case 'Data Entry Status':
+      let css='label label-default';
+      if (cell==='Complete' ) {
+        css='label label-success';
+       }
+      if (cell==='No Data Found' ) {
+        css='label label-danger';
+       }
+      if (cell==='In Progress' || cell==='Not Started') {
+        css='label label-warning';
+       }
+      result = <td><span className={css}>{cell}</span></td>;
+      break;
+    case 'Appointment Type':
+      result = <td>{row['AppointmentTypeName']}</td>;
+      break;
+    }
+
+    return result;
+}
+  renderScheduleFormButton() {
+   if (this.state.editModal) {
+     return (<ButtonElement
+            name="edit"
+            label={
+              <div>
+                <span className="glyphicon glyphicon-plus"/>Edit Appointment
+              </div>
+            }
+            type="submit"
+            buttonClass="btn btn-sm btn-success"
+           />
+     );
+    } else {
+        return (<ButtonElement
+            name="create"
+            label={
+              <div>
+                <span className="glyphicon glyphicon-plus"/>Create Appointment
+              </div>
+            }
+            type="submit"
+            buttonClass="btn btn-sm btn-success"
+           />
+         );
+    }
+  }
+  renderScheduleForm() {
+  let year = new Date();
+  let minYear = year.getFullYear();
+    const title = this.state.editModal ? 'Edit Appointment' : 'Add Appointment';
+    return (
+      <Modal
+        title= {title}
+        onClose={this.closeModal}
+        show={this.state.showModal}
+      >
+        <FormElement
+          Module="schedule"
+          name="addSchedule"
+          id="addScheduleForm"
+          onSubmit={this.handleSubmit}
+          method="POST"
+        >
+          <TextboxElement
+            name="DCCID"
+            label="DCCID"
+            value={this.state.formData.DCCID}
+            required={true}
+            onUserInput={this.setFormData}
+            disabled={this.state.editModal}
+          />
+          <TextboxElement
+            name="PSCID"
+            label="PSCID"
+            value={this.state.formData.PSCID}
+            required={true}
+            onUserInput={this.setFormData}
+            disabled={this.state.editModal}
+          />
+          <SelectElement
+            name="Session"
+            options={this.state.formData.SessionFieldOptions}
+            label="Session"
```

Comment:
```suggestion
            label="Visit"
```
I think the more familiar label for timepoint that is more consistent across Loris is 'Visit'